Improve readability of Introduction section

bwendling opened the pull request:
Hopefully this flows a bit better.

- Reworded some passive phrases.
- Combined similar sentences into single paragraphs.
- Removed some duplicate text that was repeated later.
- Some grammar-o's.

kbeyls reviewed (changes requested):
Thank you so much for this, @bwendling !
I think this looks great.

The only change I'm not sure of is the use of "There're" instead of "There are". It's just that I haven't seen "There're" much (or at all?) in the English texts I've read as a non-native English speaker. I'm happy to take those changes too though, if you think "There're" is better.

If you could also add the SPDX-FileCopyrightText line as commented on inline, this LGTM!

> *[Reviewable](https://reviewable.io/reviews/llsoftsec/llsoftsecbook/191#-:-NQUET3k1_mLu8V2ta1Z:bqrncci)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @bwendling)

___
*[`book.md` line 4 at r1](https://reviewable.io/reviews/llsoftsec/llsoftsecbook/191#-NQUFUMF6BOLYT-H743t:-NQUFUMG3SdzncSJf4bA:b6sajhi) ([raw file](https://github.com/llsoftsec/llsoftsecbook/blob/a932e4b54359e5b6e75dce207f1cf20c2264f5b8/book.md#L4)):*
> ```Markdown
> SPDX-License-Identifier: CC-BY-4.0
> copyright:
>   - SPDX-FileCopyrightText: Copyright 2021-2022 Arm Limited <open-source-office@arm.com>
> ```

I think it would be good to add a copyright line covering your contribution here too, per our contribution guidelines at https://github.com/llsoftsec/llsoftsecbook/blob/main/contributing.md#copyright-and-license.

If this is a personal contribution, I assume the line could look something like:
```
- SPDX-FileCopyrightText: Copyright 2023 Bill Wendling <contact info, such as an email address>
```

I understand if you'd prefer not to put an email address for contact info or simply not put any contact info there.


<!-- Sent from Reviewable.io -->

bwendling commented:
Hi @kbeyls, "there're" is probably not as widely used as "there are" so I changed it back. Also added the copyright.

I think I did this correctly. :-)
